bigtable-dataflow-parent/bigtable-hbase-dataflow/src/main/java/com/google/cloud/bigtable/dataflow/CloudBigtableConfiguration.java

```diff
@@ -253,6 +253,9 @@ public BigtableOptions toBigtableOptions() throws IOException {
   public Configuration toHBaseConfig() {
     Configuration config = new Configuration(false);
 
+    // Dataflow should use a different endpoint for data operations than online traffic.
+    config.set(BigtableOptionsFactory.BIGTABLE_HOST_KEY, "batch-bigtable.googleapis.com");
```

Comment:
Dataflow works with programmatic configuration, so we have to write code rather than XML.

The point of this PR is to use a different endpoint for google.bigtable.endpoint.host.  batch-bigtable.googleapis.com will be for Dataflow and other batch processes, and bigtable.googleapis.com will be for user's online operations (i.e. traffic from live serving systems).